### PR TITLE
Save memory (move usb mass storage buffer to reusableBuffer)

### PR DESCRIPTION
**Pull request by schwabe**
I still feel guilty about giving X9D+ builds about 4k less memory per default as USB mass storage is always included. This commit does

- Lower that buffer to 512 bytes per default on F2 targets, makes usb speed slightly slower ~600 kB/s instead ~700 KB/s in my tests, saving 3584 bytes
- Save another ~300 bytes by putting that buffer into reusableBuffer
- Put a 128byte variable on stack that was global but does not need to be

### CODE REVIEW

**Comment by raphaelcoeffic** [comment]
Thx! That’s going to help a lot of people fighting with lua & memory!

File: radio/src/mixer.cpp [hunk 1]
```diff
@@ -1109,6 +1108,7 @@ uint8_t   flightModeTransitionLast = 255;
 
 void evalMixes(uint8_t tick10ms)
 {
+  int32_t sum_chans512[MAX_OUTPUT_CHANNELS] = {0};
```
Comment (bsongis) [inline radio/src/mixer.cpp:1111]: The initialization seems not needed as it is done below (when fadde is in use)

**Comment by bsongis** [comment]
I will only remove the unneeded initialization in case it increases flash usage on 9X

**Comment by bsongis** [comment]
If I am not mistaken it uses 26bytes for nothing

**Pull request merged**